ethers-contract/Cargo.toml

```diff
@@ -23,9 +23,8 @@ rustdoc-args = ["--cfg", "docsrs"]
 all-features = true
 
 [dependencies]
-ethers-providers.workspace = true
+ethers-providers = { workspace = true, optional = true }
 ethers-core.workspace = true
-ethers-signers.workspace = true
```

Comment:
Is this not needed anymore?

Reply:
No, it is only used in `tests/it/abigen.rs`